lib/functions-custom.php

```diff
@@ -19,17 +19,30 @@ function nm_get_support_autovalues() {
         $key = false;
       }
 
-      $values = array(
-        isset($autovalues_set['regular_low']) ? $autovalues_set['regular_low'] : null,
-        isset($autovalues_set['regular_medium']) ? $autovalues_set['regular_medium'] : null,
-        isset($autovalues_set['regular_high']) ? $autovalues_set['regular_high'] : null,
-        isset($autovalues_set['oneoff_low']) ? $autovalues_set['oneoff_low'] : null,
-        isset($autovalues_set['oneoff_medium']) ? $autovalues_set['oneoff_medium'] : null,
-        isset($autovalues_set['oneoff_high']) ? $autovalues_set['oneoff_high'] : null,
-      );
-
-      if ($key && count(array_filter($values, 'is_numeric')) === 6) {
-        $return[$key] = $values;
+      $data = new \stdClass;
+
+      // probably better here to check type and return blank with any failure
+
+      $data->show_first = isset($autovalues_set['show_first']) ? $autovalues_set['show_first'] : null;
```

Comment:
update from dev and add required validation to these metafields as to reduct potential fragility that these changes introduce. OR should this all be validated here in a similar way to before? OR both?